DailyReportWeb/Models/PolicySummary.cs

```diff
@@ -7,8 +7,11 @@ namespace DailyReportWeb.Models
 {
     public class PolicySummary
     {
-        public string Id { get; set; }
-        public string ProjectName { get; set; }
-        public string ReportTime { get; set; } 
+        public string id { get; set; }
+        public string key { get; set; }
+        public string name { get; set; }
+        public string time { get; set; }
+        public string baseUrl { get; set; }
+        public string sharedSecret { get; set; }
```

Comment:
Astea nu sunt ok. Trebuie puse inapoi cu litera mare. 

Insa trebuie sa pui in RouteConfig:

`            var jsonFormatter = GlobalConfiguration.Configuration.Formatters.OfType<JsonMediaTypeFormatter>().First();
            jsonFormatter.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();`